packages/graph/src/graph2/graphReact.ts

```diff
@@ -76,9 +76,9 @@ export class Graph2 extends GraphReactT<ISubgraph, IVertex, IEdge> {
         this._centroidFilter.update(this.centroidColor());
     }
 }
-Graph2.prototype._class += " graph_Graph2";
+GraphReact.prototype._class += " graph_React";
```

Comment:
Should this be `graph_GraphReact`? Github Actions is noting some test failures below.